### PR TITLE
Adding CaratLane app to react-native showcase

### PR DESCRIPTION
**Pull request by anishakd4**
CaratLane app is using react-native in HomePage. We recently went live in react-native and happy to share this news with the community.

CaratLane is one of the leading online jewellery company.

### CODE REVIEW

File: website/src/react-native/showcase.js [hunk 1]
```diff
@@ -149,8 +149,14 @@ var apps = [
...
   {
-    name: 'CBS Sports Franchise Football',
+    name: 'CaratLane',
     icon: 'http://a2.mzstatic.com/us/r30/Purple69/v4/7b/0c/a0/7b0ca007-885a-7cfc-9fa2-2ec4394c2ecc/icon175x175.png',
+    link: 'https://play.google.com/store/apps/details?id=com.caratlane.android&hl=en',
+    author: 'CaratLane',
+  },
+  {
+    name: 'CBS Sports Franchise Football',
+    icon: 'https://lh3.googleusercontent.com/wEN-Vvpbnw_n89dbXPxWkNnXB7sALKBKvpX_hbzrWbuC4tFi5tVkWHq8k5TAvdbf5UQ=w300-rw',
```
Comment (chirag04) [inline website/src/react-native/showcase.js]: seems like you swapped the icon link.

Comment (anishakd4) [inline website/src/react-native/showcase.js]: Yeah by mistake I swapped the links. I rectified and updated the pull request

**Comment by anishakd4** [comment]
But still in website "CaratLane" is not reflecting in the react-native showcase.

**Comment by chirag04** [comment]
@anishakd4  website will be deployed again in some time.

**Pull request merged**